Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:10.15.3
```

Comment:
Shouldn't there be additional commands to install the app dependencies inside the container? Or is this step done after you install them outside the container? This should be documented either way

Reply:
you are right.
I've moved the yarn setup & yarn build commands into the dockerfile.
now when building the image, all of the files are automatically generated